lib/video_thumbnail.dart

```diff
@@ -27,7 +27,7 @@ class VideoThumbnail {
   static Future<String?> thumbnailFile(
       {required String video,
       Map<String, String>? headers,
-      String? thumbnailPath,
+      required String thumbnailPath,
```

Comment:
Any reason why this is now required?

Reply:
Hi, I think you can skip this change for now. The modifications in build.gradle are essential.